lib/cli/src/Cardano/CLI.hs

```diff
@@ -1318,15 +1318,16 @@ tlsOption = TlsConfiguration
         <> metavar "FILE"
         <> help "The RSA Server key which signed the x.509 server certificate."
 
-smashURLOption
+poolMetadataSourceOption
     :: Parser PoolMetadataSource
-smashURLOption = option (eitherReader reader) $ mempty
+poolMetadataSourceOption = option (eitherReader reader) $ mempty
     <> long "pool-metadata-fetching"
-    <> metavar "STRATEGY"
-    <> help "Pool Metadata fetching strategy. This setting will persist across restarts. Possible values: \
-            \ none, \
-            \ direct, \
-            \ <SMASH-URL>."
+    <> metavar "( none | direct | SMASH-URL )"
+    <> help ("Stake pool metadata fetching strategy. "
+            <> "Provide a URL to specify a SMASH metadata proxy server, "
+            <> "use \"direct\" to fetch directly from the registered pool URLs,"
+            <> " or \"none\" to completely disable stake pool metadata. "
+            <> "This setting will persist across restarts.")
```

Comment:
Perhaps: "This setting will persist across restarts AND override any previous setting set via this option or via the API." ?

Reply:
I'm starting to think it was a bad idea to keep the cli switch. It's confusing. Now someone might still think it **pins** the setting or something.

Reply:
I think it's fine to have a sticky setting like this. Makes sense to me. Anyway, our users are likely to _either_ configure this by the CLI, _or_ by API, but not both.

Reworded a bit more.